src/db/schema.rs

```diff
@@ -0,0 +1,24 @@
+table! {
```

Comment:
This file is automatically generated from `diesel.toml` or the other way around?

Reply:
This is autogenerated from diesel indeed.